pubspec.yaml

```diff
@@ -16,8 +16,8 @@ environment:
 dependencies:
   flutter:
     sdk: flutter
-  http: ^0.13.0
-  shared_preferences: ^2.0.0
+  http: ">=1.0.0 <1.2.0"
+  shared_preferences: ^2.2.0
```

Comment:
2.1.0 made the minimum flutter sdk version required to be 3.3. I'd still like to support SDK version 3.0. maybe we make this `>=2.0.0 <3.0.0`?

Reply:
versions have been updated @nkshah2 👍